app/Command/Compile.hs

```diff
@@ -12,10 +12,28 @@ import Data.Map qualified as M
 import Data.Set qualified as S
 import Data.Text qualified as T
 import Data.Traversable (for)
-import Language.PureScript qualified as P
+import Language.PureScript.AST.Declarations qualified as P
+    ( getModuleName )
```

Comment:
We should either qualify or list explicitly, but not both at once, right?

Reply:
Right. Hmm... This is going to take a lot more work then.

Reply:
I'm going to prefer to list explicitly for now (unless there are name conflicts) because having multiple modules imported qualified under the same alias still doesn't help me know which one imports the identifier I care about.
If we always want a module imported qualified, we can do a later pass.

Reply:
Could we instead leave anything that's currently qualified alone for now—not resolve the multi-module-alias ambiguities in this PR? Not only is that less immediate work, I suspect the resolution to those cases will merit more discussion than the rest and it might be best to merge the uncontroversial stuff first.

Reply:
So... Basically leave it as `import Foo qualified as Bar (baz)`?

Reply:
Or do you mean leave `import Foo as Foo` and don't expose the import list at all?

Reply:
I meant leave it as `import Foo qualified as Bar` if that's what it is approximately, just for this pass.

I think I have a good way to do the uncontroversial changes (unqualified open imports gaining explicit import lists, nicely formatted and sorted) with automation, so I'm going to submit a PR with the result of that and then the remaining cases (which should just be this multi-module-alias issue) we can hammer out manually together.